src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int INITIAL_CAPACITY = 16;
+    public static final int GROW_VALUE = 2;
+    public static final float LOAD_FACTOR = 0.75f;
```

Comment:
```suggestion
    public static final int GROW_FACTOR = 2;
    public static final float LOAD_FACTOR = 0.75f;
```
let's uniform the naming here